app/views/users/index.html.erb

```diff
@@ -1,2 +1,9 @@
-<h1>Users#index</h1>
```

Comment:
Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB34BLWTYJJHKCGPEGXPRQG4Y4JF7UA) wireframe for user's index page. The layout of the page should be the same as in the given wireframe.